### PR TITLE
[miio] update readme

### PR DESCRIPTION
**Pull request by marcelrv**
Update readme 
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.miio/README.md [hunk 1]
```diff
@@ -926,6 +927,15 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
 | led              | Switch  | Wifi led                            |
 
 
+### Mi Power-plug (<a name="chuangmi-plug-m3">chuangmi.plug.m3</a>) Channels
+
```
Comment (Confectrician) [inline addons/binding/org.openhab.binding.miio/README.md]: This line could be removed.

File: addons/binding/org.openhab.binding.miio/README.md [hunk 2]
```diff
@@ -926,6 +927,15 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
...
+
+| Channel          | Type    | Description                         |
+|------------------|---------|-------------------------------------|
+| power            | Switch  | Power                               |
+| temperature      | Number  | Temperature                         |
+
```
Comment (Confectrician) [inline addons/binding/org.openhab.binding.miio/README.md]: This line could be removed.

File: addons/binding/org.openhab.binding.miio/README.md [hunk 3]
```diff
@@ -2173,6 +2183,16 @@ Number temperature "Temperature" (G_plug) {channel="miio:basic:plug:temperature"
 Switch led "Wifi led" (G_plug) {channel="miio:basic:plug:led"}
 ```
 
+### Mi Power-plug (chuangmi.plug.m3) item file lines
+
```
Comment (Confectrician) [inline addons/binding/org.openhab.binding.miio/README.md]: This line could be removed.

File: addons/binding/org.openhab.binding.miio/README.md [hunk 4]
```diff
@@ -2173,6 +2183,16 @@ Number temperature "Temperature" (G_plug) {channel="miio:basic:plug:temperature"
...
+
+note: Autogenerated example. Replace the id (plug) in the channel with your own. Replace `basic` with `generic` in the thing UID depending on how your thing was discovered.
+```
```
Comment (Confectrician) [inline addons/binding/org.openhab.binding.miio/README.md]: Please add a newline before the codeblock and add a language like ` ```java`

**Comment by marcelrv** [comment]
@Confectrician are you 100% sure that without these blank lines it will work with Jekyll. Before it caused some issues...
https://github.com/openhab/openhab2-addons/issues/956

**Comment by Confectrician** [comment]
We don't use jekyll anymore for the website build, so keeping just one newline should be fine.

I am sure the code blocks would even work without any newline too, but the empty lines between a paragraph and codeblocks/headlines, have evolved to a kind of styleguide now.
So the reason for suggesting this is, to keep the files consistenst (and hopefully get rid of inconsistent ones in a future far away).

I have asked the other documentation maintainers how we have to (or should) handle the code block language tags in the future.
I think vupress (our current webstie/docs backend) doesn't need a special language tag for the codeblocks, but again we should try to be consistent here.
So i might suggested the wrong language in this case but that should be no problem for now.

**Comment by marcelrv** [comment]
as your remarks basically apply to the rest of the readme as well it is updated for the other items as well

**Pull request merged**